sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,218 @@
+# May'21 release notes
+
+## New features
```

Comment:
I suggest adding a note:
- [ESIMD] Allowed ESIMD and regular SYCL kernels to coexist in the same translation unit and in the same program. The `-fsycl-explicit-simd` option is no longer required for compiling ESIMD code and was deprecated. DPCPP RT implicitly appends `-vc-codegen` compile option for ESIMD images.

There were ~25 commits, so it's not possible to list them all. :)